ui/helpers/ramps/rampApi/rampAPI.ts

```diff
@@ -0,0 +1,21 @@
+import getFetchWithTimeout from '../../../../shared/modules/fetch-with-timeout';
+import { AggregatorNetwork } from '../../../ducks/ramps/types';
+
+const fetchWithTimeout = getFetchWithTimeout();
+
+const rampApiBaseUrl =
+  process.env.METAMASK_RAMP_API_CONTENT_BASE_URL ||
+  'https://on-ramp-content.api.cx.metamask.io';
```

Comment:
I have a concern here and I request advice from other reviewers: this value is pointing to production and will be used by any external collaborator when they run a local or forked version of the extension, adding extra load to our production server, should we change it to default to our dev/staging url?

Reply:
https://github.com/MetaMask/metamask-extension/pull/24041/commits/04b5c8f873fd8f48f92c93d0c4d499aa6f76a361